examples/Bunch/include/simulation_defines/param/memory.param

```diff
@@ -56,4 +56,11 @@ constexpr uint32_t BYTES_EDGES = 32 * 1024; //32 kiB;
 /** number of scalar fields that are reserved as temporary fields */
 constexpr uint32_t fieldTmpNumSlots = 1;
 
+/** can `FieldTmp` gather neighbor information
+ *
+ * If `true` it is possible to call the method `asyncCommunicationGather()`
+ * to copy data from the border of neighboring GPU into the local guard.
+ */
```

Comment:
Ahh I see you tried if I copy&past your text without reading it^^ `cyle` must be `cycle`